Flow.Launcher/SettingPages/Views/SettingsPaneTheme.xaml.cs

```diff
@@ -28,4 +31,77 @@ private void Selector_OnSelectionChanged(object sender, SelectionChangedEventArg
     {
         _viewModel.UpdateColorScheme();
     }
+
+    private void Reset_Click(object sender, System.Windows.RoutedEventArgs e)
+    {
+        /*The FamilyTypeface should initialize all of its various properties.*/
+        FamilyTypeface targetTypeface = new FamilyTypeface { Stretch = FontStretches.Normal, Weight = FontWeights.Normal, Style = FontStyles.Normal };
+
+        QueryBoxFontSize.Value = 20;
+        QueryBoxFontComboBox.SelectedIndex = SearchFontIndex("Segoe UI", QueryBoxFontComboBox);
+        QueryBoxFontStyleComboBox.SelectedIndex = SearchFontStyleIndex(targetTypeface, QueryBoxFontStyleComboBox);
+
+        ResultItemFontComboBox.SelectedIndex = SearchFontIndex("Segoe UI", ResultItemFontComboBox);
+        ResultItemFontStyleComboBox.SelectedIndex = SearchFontStyleIndex(targetTypeface, ResultItemFontStyleComboBox);
+        ResultItemFontSize.Value = 16;
+
+        ResultSubItemFontComboBox.SelectedIndex = SearchFontIndex("Segoe UI", ResultSubItemFontComboBox);
+        ResultSubItemFontStyleComboBox.SelectedIndex = SearchFontStyleIndex(targetTypeface, ResultSubItemFontStyleComboBox);
+        ResultSubItemFontSize.Value = 13;
+
+        WindowHeightValue.Value = 42;
+        ItemHeightValue.Value = 58;
+    }
```

Comment:
Initially, we worked by changing the settings value, but there was an issue with the control not reflecting the change. After a few tries, we settled on just changing the value of the control.

+) This initial value has been matched to the initial value in settings.cs and should be matched in the future. No problem if it's different. Should we replace it with a contant value and align them both to constant?